tools/walletextension/frontend/src/components/layouts/header.tsx

```diff
@@ -14,7 +14,7 @@ export default function Header() {
       <div className="flex h-16 justify-between items-center px-4">
         <Link href="/">
           <Image
-            src="/static/assets/images/black_logotype.png"
+            src="/assets/images/black_logotype.png"
```

Comment:
logos are now broken in both white and black version.
I run: npm run build and then serve static files with golang (same way as then deployed to Azure).